ui/tests/integration/components/filter-input-test.js

```diff
@@ -5,22 +5,23 @@
 
 import { module, test } from 'qunit';
 import { setupRenderingTest } from 'ember-qunit';
-import { render, fillIn } from '@ember/test-helpers';
+import { render, fillIn, click } from '@ember/test-helpers';
 import hbs from 'htmlbars-inline-precompile';
 
 module('Integration | Component | filter-input', function (hooks) {
   setupRenderingTest(hooks);
 
-  test('it should render initial value', async function (assert) {
```

Comment:
I think we still should test that the initial value is shown on initial render

Reply:
That assertion is still there just moved down to line 24. It's basically the same test but I added an assertion to check the placeholder attribute and that modifiers are set on the element as well.